src/webui/components/agent_settings_tab.py

```
@@ -24,30 +21,596 @@ def update_model_dropdown(llm_provider):
+    Returns:
+        str: HTML representation of the MCP servers
+    """
+    logger.debug("Rendering MCP servers HTML")
+    servers = get_mcp_servers_list(webui_manager)
+    logger.debug(f"Found {len(servers)} MCP servers to render")
+
+    if not servers:
+        logger.debug("No MCP servers found to render")
+        return "<div class='mcp-servers-empty'>No MCP servers configured</div>"
+
+    html = "<div class='mcp-servers-container'>"
```

Ruff (0.11.9)

Redefinition of unused `html` from line 2

(F811)